frontend/src/styles.scss

```diff
@@ -247,20 +247,6 @@ body {
 	padding-right: 20px;
 }
 
-.ant-drawer {
-	// do not modify this as this is required to hide intercom in modals
-	z-index: 1000000000000;
-}
-
-.ant-modal-root {
-	.ant-modal-mask {
-		z-index: 2000000000000 !important;
-	}
-	.ant-modal-wrap {
-		z-index: 2000000000000 !important;
-	}
-}
-
-.ant-select-dropdown {
-	z-index: 3000000000000 !important;
+.intercom-lightweight-app {
+	z-index: 980 !important;
```

Comment:
Add reasoning on why we want it to be 980 and also list component that are affecting stacking context in our base css file for reference.

Reply:
done ✅